README.md

```diff
@@ -46,6 +46,17 @@ machine 'wario' do
       :ports => 1234
       :ports => "2345:6789"
 
+      # Volumes can be one of three forms:
```

Comment:
Does this syntax come from the root docker docs?  If so, it would be nice to link to those from here if people want to do further reading.